manual/manual/refman/exten.etex

```diff
@@ -2320,3 +2320,89 @@ Empty variant type can be eliminated by refutation case of pattern matching.
 type t = |
 let f (x: t) = match x with _ -> .
 \end{caml_example*}
+
+
+\section{Alerts \label{s:alerts} }
+(Introduced in 4.08)
+
+Since OCaml 4.08, it is possible to mark components (such as value or
+type declarations) in signatures with ``alerts'' that will be reported
+when those components are referenced.  This generalizes the notion of
+``deprecated'' components which were previously reported as warning 3.
+Those alerts can be used for instance to report usage of unsafe
+features, or of features which are only available on some platforms,
+etc.
+
+Alert categories are identified by a symbolic identifier (a lowercase
+identifier, following the usual lexical rules) and an optional
+message.  The identifier is used to control which alerts are enabled,
+and which ones are turned into fatal errors.  The message is reported
+to the user when the alert is triggered (i.e. when the marked
+component is referenced).
+
+The "ocaml.alert" or "alert" attribute is used to mark components.
```

Comment:
On my first reading, I found this slightly confusing:
- `alert <name> "<message>"` is used to attach an alert to a declaration
- `alert "<settings>"` is used to control alert settings

You should specify here that it's the alert attribute _with an identifier and a string_ that is used to mark components.

Reply:
Perfect.